src/com/epam/javatrainings/classdecomposition/Pizza.java

```diff
@@ -0,0 +1,94 @@
+package com.epam.javatrainings.classdecomposition;
+
+import java.util.HashMap;
+import java.util.List;
+
+import static java.lang.System.exit;
+import static java.lang.System.in;
+
+public class Pizza {
```

Comment:
Have you tested your code? if you use Pizza class as a hashmap key you must override equals/hashcode also make Pizza class immutable